Firebase/Core/FIRLogger.m

```diff
@@ -213,6 +214,115 @@ BOOL getFIRLoggerDebugMode() {
 }
 #endif
 
+#pragma mark - Number of errors and warnings
+
+NSString *FIRLoggerPathInCachesByAppending(NSString *pathToAppend) {
+  NSArray *directories =
+      NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES);
+  if (directories.count == 0) {
+    return nil;
+  }
+  NSString *cacheDir = directories[0];
```

Comment:
To avoid crashing
NSString *cacheDir = directories.firstObject;

Reply:
Done.